app/helpers/application_helper.rb

```diff
@@ -9,25 +9,8 @@ def sortable(column, title = nil)
   class BootstrapLinkRenderer < ::WillPaginate::ActionView::LinkRenderer
```

Comment:
These two lines can go too - see line 27, there's a double end there

currently it tries to mash page_navigation_links into Bootstrap which breaks